src/pages/portfolio/index.tsx

```diff
@@ -53,5 +54,9 @@ function MobilePortfolioPage() {
 }
 
 function DesktopPortfolioPage() {
-  return 'Hi from desktop!';
+  return (
+    <div className='container mx-auto py-8'>
```

Comment:
issue (ui): even though we don't support mobile devices in portfolio page, I believe we should implement the mobile screens for every page. We have Figma for it.

suggestion: `<Grid>` and `<Display>` components from UI package might help here

<img width="338" alt="image" src="https://github.com/user-attachments/assets/c61dd835-2b25-4d12-9c16-c240cbbaa0eb" />

Reply:
Fixed the UI on mobile. Do note that on mobile this page will display the notice that it's not available on mobile, but manually switching in desktop Chrome will show the fixed mobile layout.